Makefile

```diff
@@ -293,7 +293,7 @@ docs-deploy:
 		-v $(PWD)/docs/content:/docs/docs/content \
 		-v $(PWD)/docs/mkdocs.yml:/docs/mkdocs.yml \
 		-v /tmp/docs:/tmp/docs \
-			static-web-server-docs mkdocs build
+			static-web-server-docs mkdocs build --strict
```

Comment:
I wanted to add it to the pipeline as well but there does not seem to be a job/step to build the documentation site. Which of the workflows would be appropriate to add this?

Reply:
I meant that it would be good to run `mkdocs build --strict` on every PR to ensure that changes don't break anything.

Reply:
> I meant that it would be good to run `mkdocs build --strict` on every PR to ensure that changes don't break anything.

Yes, a `docs` build check step on `devel` CI could be an option.

> Releasing in the pipeline would be helpful as well.

For releasing, I was thinking about using some extra tool like `release-plz` or similar.

> Another thing is that it could be helpful to release an image from master with a specific tag. That would allow someone to run the latest state, could be helpful to test it out before a release.

For that, a `devel` Docker image built by CI should be useful containing always the latest `master` changes.

Reply:
> Yes, a docs build check step on devel CI could be an option.

Added.

> For releasing, I was thinking about using some extra tool like release-plz or similar.

Interesting. Thanks for mentioning this tool. Through that I discovered `git-cliff` which looks quite helpful for some projects of mine.